### PR TITLE
SWARM-1711: Update hystrix to 1.5.12 to align with MP Fault Tolerance Impl

### PR DESCRIPTION
**Pull request by heiko-braun**
- [x] Have you followed the guidelines in our [Contributing](http://wildfly-swarm.io/community/contributing/) document?
- [x] Have you created a [JIRA](https://issues.jboss.org/browse/SWARM) and used it in the commit message?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wildfly-swarm/wildfly-swarm/pulls) for the same issue?
- [ ] Have you built the project locally prior to submission with `mvn clean install`?

-----
HOLD: This is for verification purposes

### CODE REVIEW

File: fractions/netflix/pom.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
     <version.netflix.commons>0.1.1</version.netflix.commons>
     <version.archaius>0.6.6</version.archaius>
     <version.ribbon>2.1.0</version.ribbon>
-    <version.hystrix>1.4.21</version.hystrix>
+    <version.hystrix>1.5.12</version.hystrix>
```
Comment (mkouba) [inline fractions/netflix/pom.xml:29]: This is not enough IMHO. `hystrix-code` depends on `HdrHistogram`.

Comment (heiko-braun) [reply fractions/netflix/pom.xml]: Can you elaborate on this?

**Comment by kenfinnigan** [comment]
In future, I'd prefer that we're testing a PR locally before submitting it as then we at least know it should pass

**Comment by heiko-braun** [comment]
The latest commit adds the `hdrHistogram` deps to the module. The changes have been verified with the CB booster. All that's left is to consolidate the hystrix version property at a shared location. But I am leaving this to @mkouba to do when the tck work is completed.

**Review by kenfinnigan — APPROVED** [review]

File: fractions/netflix/hystrix/pom.xml [hunk 2]
```diff
@@ -25,6 +25,7 @@
   <properties>
     <swarm.fraction.stability>stable</swarm.fraction.stability>
     <swarm.fraction.tags>Fault Tolerance,Netflix OSS</swarm.fraction.tags>
+    <version.hdrHistogram>2.1.9</version.hdrHistogram>
```
Comment (Ladicek) [inline fractions/netflix/hystrix/pom.xml:28]: Should be in the root `pom.xml`, together with a `<dependencyManagement>` entry, like all other dependecies?

Comment (heiko-braun) [reply fractions/netflix/hystrix/pom.xml:28]: yes, but as I said, I would leave this to @mkouba to fix together with the MP TCK suite he is working on

Comment (heiko-braun) [reply fractions/netflix/hystrix/pom.xml:28]: I was a bit eager, this is hdr dep, yes maybe you are right

Comment (heiko-braun) [reply fractions/netflix/hystrix/pom.xml:28]: not sure, @kenfinnigan what's you opinion: where should deps like this be managed?

Comment (kenfinnigan) [reply fractions/netflix/hystrix/pom.xml:28]: Currently all netflix fraction dependencies are depMan'd in https://github.com/wildfly-swarm/wildfly-swarm/blob/master/fractions/netflix/pom.xml

Maybe there?

Comment (mkouba) [reply fractions/netflix/hystrix/pom.xml:28]: +1 for netflix parent

**Pull request merged**